src/MarkovChainMonteCarlo.jl

```diff
@@ -3,307 +3,587 @@ module MarkovChainMonteCarlo
 using ..Emulators
 using ..ParameterDistributionStorage
 
-using Statistics
+import Distributions: sample # Reexport sample()
 using Distributions
-using LinearAlgebra
 using DocStringExtensions
+using LinearAlgebra
+using Printf
+using Random
+using Statistics
+
+using MCMCChains
+import AbstractMCMC: sample # Reexport sample()
+using AbstractMCMC
+import AdvancedMH
+
+export
+    EmulatorDensityModel,
+    VariableStepProposal,
+    VariableStepMHSampler,
+    MCMCProtocol,
+    EmulatorRWSampling,
+    MCMCWrapper,
+    get_stepsize,
+    set_stepsize!,
+    accept_ratio,
+    find_mcmc_step!,
+    get_posterior,
+    sample
+
+# ------------------------------------------------------------------------------------------
+# Output space transformations between original and SVD-decorrelated coordinates.
+# Redundant with what's in Emulators.jl, but need to reimplement since we don't have
+# access to obs_noise_cov
+
+"""
+    to_decorrelated(data::Array{FT, 2}, em::Emulator)
+
+Transform samples from the original (correlated) coordinate system to the SVD-decorrelated
+coordinate system used by Emulator.
+"""
+function to_decorrelated(data::Array{FT, 2}, em::Emulator{FT}) where {FT<:AbstractFloat}
```

Comment:
Since the SVD and coordinate transformations are `Emulator`'s responsibility, I'd advocate for putting it in Emulators.jl, since it could also be used by the constructor there. I added the `to_decorrelated` function because we only need to transform `obs_sample` in MCMC.

This may be too nit-picky, but check out Emulators.jl on [this scratch branch](https://github.com/tsj5/CalibrateEmulateSample.jl/blob/tsj/refactor-Emulator-SVD/src/Emulator.jl): I reorganized the standardize/svd_transform etc. functions into to_decorrelated, svd_transform_cov/from_decorrelated functions which act on means and covariances separately.

Reply:
I was wondering of actually moving the SVD out of emulator even, e.g we could pass in a `decomposition` to the emulator.

Reply:
But perhaps you are right, lets leave these here for now with the view to move them.